src/openstudio_app/OpenStudioApp.cpp

```diff
@@ -1553,14 +1553,16 @@ void OpenStudioApp::loadShoeboxModel() {
   versionTranslator.setAllowNewerVersions(false);
 
   auto filePath = resourcesPath() / toPath("ShoeboxModel/ShoeboxExample.osm");
-  boost::optional<openstudio::model::Model> model = versionTranslator.loadModel(filePath);
-  if (!model && isOpenStudioApplicationRunningFromBuildDirectory()) {
+  boost::optional<openstudio::model::Model> model_;
+  if (openstudio::filesystem::is_regular_file(filePath)) {
```

Comment:
Was this the line that prevents the crash?

Reply:
currentDocument not null.

Reply:
This specific change is mostly because I found it annoying and counterproductive to try to load a model and get a warning/error on the console like "Cannot resolve path 'xxxx', model can't be loaded".

Reply:
https://github.com/NREL/OpenStudio/blob/ba1b94b6542c1f1d4cdcdb775886852b9a82bc4f/src/model/Model.cpp#L1848